terraform/install.sh

```diff
@@ -50,6 +50,30 @@ promptForBillingAccount() {
     exit 1;
   fi
 
+ #check for permission to link account
+  found_accounts=()
+  IFS=$'\n'
+  found_all_accounts=($found_all_accounts)
+  for acc in ${found_all_accounts[@]}
+  do
+    IFS=$'\t'
+    acc=($acc)
+    IFS=$'\n'
+    test_perm=$(gcloud beta billing accounts get-iam-policy "${acc[1]}" --flatten=bindings --filter="bindings.role:roles/billing.projectManager OR bindings.role:roles/billing.user OR bindings.role:roles/billing.admin" --format='value(bindings.members)')
+    if [ -n "$test_perm" ] || [[ ${#test_perm[@]} -ne 0 ]]; then
+      found_accounts+=("${acc[1]}")
+    fi
+  done
+
+  if [[ ${#found_accounts[@]} -eq 0 ]]; then
+    log "error: you don't have permissions to link projects to billing accounts,"
+    log "Follow this link to setup a new billing account that you can link:"
+    log "https://cloud.google.com/billing/docs/how-to/manage-billing-account"
+    sendTelemetry "none" no-active-billing
+    exit 1;
+  fi
```

Comment:
nit: the case when no billing accounts found and when the authorized user does not have permissions to an account are identical. it might be useful to refactor the code and to print the same message asking to ensure that the executor has the required permissions to at least one billing account.